### PR TITLE
Update license info from OSS Tool

### PR DESCRIPTION
**Pull request by bhavyaus**

### CODE REVIEW

**Review by Tyriar — DISMISSED** [review]

**Review by Tyriar — APPROVED** [review]

**Review by Tyriar — APPROVED** [review]

File: cli/ThirdPartyNotices.txt [hunk 1]
```diff
@@ -3366,7 +3366,7 @@ DEALINGS IN THE SOFTWARE.
 
 hermit-abi 0.1.19 - MIT/Apache-2.0
 hermit-abi 0.3.1 - MIT OR Apache-2.0
-https://github.com/hermitcore/rusty-hermit
+https://github.com/hermit-os/hermit-rs
```
Comment (Tankdeebs) [inline cli/ThirdPartyNotices.txt:3369]: + https://github.com/hermit-os/hermit-rs

File: cli/ThirdPartyNotices.txt [hunk 2]
```diff
@@ -2112,7 +2112,7 @@ SOFTWARE.
 ---------------------------------------------------------
 
 dialoguer 0.10.4 - MIT
-https://github.com/mitsuhiko/dialoguer
+https://github.com/console-rs/dialoguer
```
Comment (Tankdeebs) [inline cli/ThirdPartyNotices.txt:2115]: + https://github.com/console-rs/dialoguer

File: ThirdPartyNotices.txt [hunk 3]
```diff
@@ -2246,7 +2246,7 @@ SOFTWARE.
 
 ---------------------------------------------------------
 
-redhat-developer/vscode-java 1.21.0 - MIT
+redhat-developer/vscode-java 1.22.0 - MIT
```
Comment (Tankdeebs) [inline ThirdPartyNotices.txt:2249]: + redhat-developer/vscode-java 1.22.0 - MIT
+

File: package.json [hunk 4]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "code-oss-dev",
   "version": "1.83.0",
-  "distro": "51280fc5e9f5acdf5660e27d6b77f53aaad386cf",
+  "distro": "076f0e1cc9474c5c58d562af4d58c7927a616b29",
```
Comment (Tankdeebs) [inline package.json:4]: + distro": "076f0e1cc9474c5c58d562af4d58c7927a616b29

File: ThirdPartyNotices.txt [hunk 5]
```diff
@@ -1563,7 +1563,7 @@ SOFTWARE.
 
 ---------------------------------------------------------
 
-jlelong/vscode-latex-basics 1.5.3 - MIT
+jlelong/vscode-latex-basics 1.5.2 - MIT
```
Comment (Tankdeebs) [inline ThirdPartyNotices.txt:1566]: + jlelong/vscode-latex-basics 1.5.2 - MIT

**Pull request merged**